.devcontainer/nats/resolver.conf

```diff
@@ -0,0 +1,15 @@
+// Operator "LOCAL"
```

Comment:
You shouldn't need this, its generated by the `nats_init.sh` script you have added